### ISSUE TITLE
KB-divergence in sequential analysis

### ISSUE THREAD

**Issue by janderuiter** [jasp-stats/jasp-desktop#847]
I think it would be nice and very useful in the sequential analysis to not only have a plot of the BF and how it developes over time while the data come in, but also the Kullback-Leibler distance between prior before datapoint N comes in, and the posterior after datapoint N has been processed. This gives a measure of how informative or 'surprising' incoming datapoints are. Normally, this KB measure would be expected to asymptotically approach zero with increasing number of data points, but deviations from this pattern are usually interesting and important.

https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence

**Comment by jonathon-love** [comment]
let me attend @EJWagenmakers to this.

**Comment by EJWagenmakers** [comment]
Yes that's interesting. An additional possibility is to compute the informativeness for each point separately; so we assess the informativeness of observation x_i by computing the KL change that it brings about compared to the situation when all other data are in. As an aside, the BF is the ratio of prior and posterior height at delta=0, so the change in the BF is the change in the posterior at delta=0; therefore I think it will show some similar patterns to KL.
E.J.

**Comment by janderuiter** [comment]
Computing the informativeness for each point separately is a great idea! I
was thinking of how to present these measures visually, and thought it
might be nice to compute it for all datapoints (so for every datapoint D
compute KL[data without D, all_data]) and then plot these measures in the
same graph as in the sequential BF plot. This way the user can see at a
glance *which* datapoint was high or low on surprise value.

**Comment by janderuiter** [comment]
Hi guys,

Is there any news/decision on the suggestion of implementing the
KL-distance in JASP plots?

Best, JP

**Comment by EJWagenmakers** [comment]
It's on the to-do list. That is a long list, but we will get to it
eventually. We will apply for more money soon so that might help
E.J.

********************************************
Eric-Jan Wagenmakers
Department of Psychological Methods, room 2.09
University of Amsterdam
Weesperplein 4
1018 XA Amsterdam
The Netherlands

Web: ejwagenmakers.com
Book: bayesmodels.com
Stats: jasp-stats.org
Email: EJ.Wagenmakers@gmail.com
Phone: (+31) 20 525 6420

“Man follows only phantoms.”
Pierre-Simon Laplace, last words
********************************************

**Comment by janderuiter** [comment]
Cool. Thanks for the quick response, and good luck with the list!

JP

**Comment by janderuiter** [comment]
Hi! Does that mean you implemented this? *Jumping up and down
enthusiastically*

Cheers, JP

**Comment by EJWagenmakers** [comment]
Nee JP
Bruno heeft het gekilled omdat het te lang geleden is dat we er iets mee
hebben gedaan denk ik :-) Dus het was vanwege een grote schoonmaak.
Blijf ons lastigvallen hiermee svp.
E.J.

********************************************
Eric-Jan Wagenmakers
Department of Psychological Methods, room G 0.29
University of Amsterdam, Nieuwe Achtergracht 129B
Letter: PO Box 15906, 1001 NK Amsterdam
Parcel: Valckenierstraat 59,  1018 XE Amsterdam

Web: ejwagenmakers.com
Book: bayesmodels.com
Stats: jasp-stats.org
Email: EJ.Wagenmakers@gmail.com
Cell: (+31) 6 45626624

“Man follows only phantoms.”
Pierre-Simon Laplace, last words
********************************************

**Comment by janderuiter** [comment]
Jammer! Het zou zo cool zijn!

**Comment by boutinb** [comment]
Sorry te snel ge-closed....
